check_source.py

```diff
@@ -62,6 +62,8 @@ def target_project_config(self, project):
         self.allow_valid_source_origin = str2bool(config.get('check-source-allow-valid-source-origin', 'False'))
         self.valid_source_origins = set(config.get('check-source-valid-source-origins', '').split(' '))
         self.add_devel_project_review = str2bool(config.get('check-source-add-devel-project-review', 'False'))
+        self.allowed_scm_submission_sources = set(
+            config.get('allowed-scm-submission-sources', 'devel:Factory:git-workflow:staging').split(' '))
```

Comment:
why does it needs a hardcoded value? add an config-bit to OSRT:Config that implied you want to have a flexible configurations, `devel:Factory:git-workflow:staging` can be set on the target project instead of to hardcode it here.

Reply:
What you are asking for is what the code should be doing. `devel:Factory:git-workflow:staging` is just the default value if no config is set.

Reply:
IMO `devel:Factory` only makes sense for `openSUSE:Factory` so the default here should be empty